rust/private/rustc.bzl

```diff
@@ -440,7 +441,14 @@ def construct_arguments(
     args.add_all(rust_lib_paths, before_each = "-L", format_each = "%s")
 
     args.add_all(rust_flags)
-    args.add_all(getattr(attr, "rustc_flags", []))
+    args.add_all(
+        expand_arg_locations(
+            ctx,
+            getattr(attr, "rustc_flags", []),
+            getattr(attr, "data", []) +
+            getattr(attr, "compile_data", []),
```

Comment:
Nit: Could you put these 2 lines (data and compile_data) on the same line so future me realizes they are added to create a third argument to the expand_arg_locations (if buildifier doesn't allow that, please extract a local variable)? I just starred at this for few minutes not understanding what's going on because I missed that there is a comma at the end of the line 447, and there is a plus on 448 :) Thank you!